.github/workflows/laravel.yml

```diff
@@ -3,12 +3,11 @@ name: Laravel CI
 on: [pull_request]
 
 jobs:
-  run:
+  build:
```

Comment:
I agree. Nevertheless, this can introduce a new set of changes. I believe none of the code exchange tutorials I've worked on have linters. What do you think if I include it in another separate PR instead of blocking this and the other ones?

Reply:
sound good!